src/pages/blog/_markets_news_data.js

```diff
@@ -0,0 +1,50 @@
+export const market_news_data = [
+    {
+        report_type: "Weekly report",
```

Comment:
Could you add a few more objects as well in between with different report_types? And we will need to filter on the FE to only show the ones where report_type === 'Weekly report'